src/browser_action/components/BrowserActionApp.jsx

```diff
@@ -82,9 +85,40 @@ export default class BrowserActionApp extends React.Component {
     window.close();
   }
 
+  /**
+   * Show the Study popup when the StudyFooter is clicked
+   */
+  handleClickStudy() {
+    this.setState({showStudyInvitation: true});
+  }
+
+  /**
+   * Return to the TrackedProductList when the back button on the Study popup is clicked
+   */
+  handleClickBack() {
+    this.setState({showStudyInvitation: false});
+  }
+
+  /**
+   * Open the Study recruitment survey page and close the panel when the participate button
+   * in the Study popup is clicked
+   */
+  handleClickParticipate() {
+    browser.tabs.create({url: 'https://www.surveygizmo.com/s3/4693160/Price-Wise-Research-Study-Participant-Screener'});
+    window.close();
+  }
+
   render() {
     const {products} = this.props;
-    const {extractedProduct} = this.state;
+    const {extractedProduct, showStudyInvitation} = this.state;
+    if (showStudyInvitation) {
```

Comment:
Can we add a config check here so that we can quick-disable this when it's time to stop recruiting? We can put up a PR changing the config default and just deploy that, and then remove the code itself later without the pressure.

Reply:
I added this feature flag check in the `<StudyFooter />` component, which gates the `showStudyInvitation` value.